app/scripts/modules/core/delivery/details/singleExecutionDetails.controller.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.singleExecutionDetails.controller', [
+    require('angular-ui-router'),
+    require('../service/execution.service.js'),
+    require('../../scheduler/scheduler.factory.js'),
+
+  ])
+  .controller('SingleExecutionDetailsCtrl', function ($scope, $state, executionService, schedulerFactory) {
+
+    let executionScheduler = schedulerFactory.createScheduler(5000);
```

Comment:
@anotherchrisberry set scheduler interval to 5000. Wonder if this should be something inside of settings.js